thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datalayer/pojo/DetectionConfigBean.java

```diff
@@ -125,6 +127,14 @@ public void setActive(boolean active) {
     this.active = active;
   }
 
+  public List<String> getLixFlags() {
```

Comment:
Why this is called `LixFlags`? This seems wired to me. `Lix` is an internal concept and shouldn't be exposed to the opensource project. 

Also, looks like we are using it only for availability schedule for now, why not use a boolean value and call it something like`isEventDrivenSchedule`, which is more self-explanatory.